### PR TITLE
Switch to Pulumi blog post

### PR DESCRIPTION
**Pull request by aaronkao**
New blog post that is part of TF Takeover campaign

### CODE REVIEW

File: content/blog/why-switch-to-pulumi/index.md [hunk 1]
```diff
@@ -0,0 +1,470 @@
+---
+title: "Why Switch to Pulumi for Cloud Deployments?"
+date: 2024-07-17T19:47:50-07:00
```
Comment (interurban) [inline content/blog/why-switch-to-pulumi/index.md:3]: update date before publishing

File: content/blog/why-switch-to-pulumi/index.md [hunk 2]
```diff
@@ -0,0 +1,470 @@
...
+change and direction the industry is going, so you never fall
+behind with your IaC tooling.
+
+**Powering the Next Wave.** Pulumi is at the forefront of the industry.
+It helps you embrace the latest practices (e.g., Platform Engineering, GitOps) and
+builds the latest technologies (e.g., AI Copilots) into the core user
+experience. It also helps you build and manage new technology
+paradigms (e.g., AI workloads, LLMs, internal developer platforms).
+
+**Powerful Automations.** Another unique approach is Pulumi makes it
```
Comment (interurban) [inline content/blog/why-switch-to-pulumi/index.md:85]: Suggestion: Another unique approach is Pulumi, which makes it easy to automate and scale modern cloud architectures.

File: content/blog/why-switch-to-pulumi/index.md [hunk 3]
```diff
@@ -0,0 +1,470 @@
...
+altered their licensing and introduced uncertainty. They are no longer
+true open source, and they tie their previously open source software to
+their commercial services. The lack of an open source approach fragments
+the community and introduces proprietary constraints to cloud
+infrastructure.\
+\
+*Pulumi provides you with an approach to open source and community that
+provides stability and choice.*
+
+**Increased Productivity:** You expect your tools
```
Comment (interurban) [inline content/blog/why-switch-to-pulumi/index.md:131]:
```suggestion
**Increased Productivity.** You expect your tools
```

File: content/blog/why-switch-to-pulumi/index.md [hunk 4]
```diff
@@ -0,0 +1,470 @@
...
+
+{{% notes type="info" %}}
+Customers of Pulumi frequently experience 100% productivity increases, 99% time saved, and 10x acceleration when using Pulumi compared to what they were using before.
+
+![modern-benefits](modern-benefits.png)
+{{% /notes %}}
+
+## Frequently Asked Questions - Why Pulumi?
+
+Some frequently asked questions about why Pulumi based on use cases and
```
Comment (interurban) [inline content/blog/why-switch-to-pulumi/index.md:160]: suggestion: Here are some frequently asked questions about why you should choose Pulumi based on the following use cases and alternatives.

File: content/blog/why-switch-to-pulumi/index.md [hunk 5]
```diff
@@ -0,0 +1,470 @@
...
+There are many different use cases for why you might use Pulumi:
+infrastructure CICD, modern applications, internal developer platforms,
+AI/ML workloads, and infrastructure modernization. Below are more
+details on two popular use cases.
+
+#### Why Pulumi for Internal Developer Platforms?
+
+You and your team may build internal developer platforms (IDPs) to
+maximize the use of the cloud at scale across the company while
+being secure and compliant so your development teams can ship faster. There are a number of components/layers that are considered basic requirements when building an internal
```
Comment (interurban) [inline content/blog/why-switch-to-pulumi/index.md:184]:
```suggestion
being secure and compliant, so your development teams can ship faster. There are a number of components/layers that are considered basic requirements when building an internal
```

File: content/blog/why-switch-to-pulumi/index.md [hunk 6]
```diff
@@ -0,0 +1,470 @@
...
+developer platform. The layers are as follows:
+
+* **Developer Control Plane.** Curated experiences that empower
+    developers by meeting them at their level of expertise, whether
+    it\'s an abstracted developer portal, custom CLI, or shared IaC
+    templates.
+
+* **Integration & Delivery.** Automations to version control, test,
```
Comment (interurban) [inline content/blog/why-switch-to-pulumi/index.md:192]: automations or automation ?

File: content/blog/why-switch-to-pulumi/index.md [hunk 7]
```diff
@@ -0,0 +1,470 @@
...
+    It\'s just harder to bring together platform, development, and
+    security teams and empower them with tools that work. Whereas Pulumi
+    unblocks infrastructure as the bottleneck for software delivery.
+
+[Read more >>](/docs/concepts/vs/terraform/)
+
+#### Why Pulumi vs. AWS Cloud Development Kit (CDK)?
+
+Pulumi and CDK are similar in that both allow you to use programming
+languages to write infrastructure as code. However there are some key
```
Comment (interurban) [inline content/blog/why-switch-to-pulumi/index.md:380]:
```suggestion
languages to write infrastructure as code. However, there are some key
```

File: content/blog/why-switch-to-pulumi/index.md [hunk 8]
```diff
@@ -0,0 +1,470 @@
...
+    contrast, CDK only permits assertions against synthesized
+    CloudFormation templates and lacks offline testing capabilities.
+    This makes Pulumi\'s testing approach more comprehensive and
+    flexible than CDK\'s.
+
+[Read more >>](/docs/concepts/vs/cloud-template-transpilers/aws-cdk/)
+
+#### Why Pulumi vs. 3rd party internal developer portal providers?
+
+There are many third party providers of internal developer portals (think: Port, Cortex, Backstage).
```
Comment (interurban) [inline content/blog/why-switch-to-pulumi/index.md:416]:
```suggestion
There are many third party providers of internal developer portals (think Port, Cortex, Backstage).
```

**Review by interurban — APPROVED** [review]
A few suggestions and an approval for when you are ready to merge.

**Comment by aaronkao** [comment]
Approval from Eli before her PTO 
image

**Pull request merged**